src/ray/raylet/worker_pool.cc

```diff
@@ -364,6 +362,29 @@ std::tuple<Process, StartupToken> WorkerPool::StartWorkerProcess(
                                   std::to_string(worker_startup_token_counter_));
   }
 
+  if (language == Language::PYTHON || language == Language::JAVA) {
```

Comment:
This just moves the block of code so `worker_command_args` handling logic are located together. Although this is unrelated to updating niceness, I prefer to keep this cleanup.